### ISSUE TITLE
Is this meant to be dark only?

### ISSUE THREAD

**Issue by SNDST00M** [OzakIOne/markdown-github-dark#4]
I can open a PR to add dark-only styling if that's welcome.
That'll greatly reduce the size of the CSS as well.

**Comment by OzakIOne** [comment]
Yeah this will probably stay dark only unless I find something interesting to have a white version.
